ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-aro.yaml

```diff
@@ -32,15 +32,10 @@ base_images:
     namespace: ocp
     tag: tools
   upi-installer:
-    name: "4.10"
+    name: "4.11"
     namespace: ocp
     tag: upi-installer
 releases:
-  intermediate410:
-    release:
-      architecture: amd64
-      channel: fast
-      version: "4.10"
   latest:
```

Comment:
The `latest` is pointing to 4.10

Reply:
Switched it to 4.11 now.. From what I recall in working on these upgrade jobs in the past we have to have a release:latest otherwise the job doesn't work. I can't recall if it is the same case for release:target so they now both point to the same release.

Reply:
The same issue exist for the other aro upgrade files

Reply:
Never mind. We can do it in a follow up if it's needed.